schemas/dab.draft.schema.json

```diff
@@ -239,6 +239,63 @@
             }
           }
         },
+        "mcp": {
+          "type": "object",
+          "description": "Global MCP endpoint configuration",
+          "additionalProperties": false,
+          "properties": {
+            "path": {
+              "default": "/mcp",
+              "type": "string"
+            },
+            "enabled": {
+              "type": "boolean",
+              "description": "Allow enabling/disabling MCP requests for all entities.",
+              "default": false
+            },
+            "dml-tools": {
+              "type": "object",
+              "description": "Global DML tools configuration",
+              "properties": {
+                "enabled": {
+                  "type": "boolean",
+                  "description": "Allow enabling/disabling DML tools for all entities.",
+                  "default": false
+                },
+                "describe-entities": {
+                  "type": "boolean",
```

Comment:
would we ever need this tool to have subproperties? If yes, it should be defined as an object here instead of a boolean.